README.md

```diff
@@ -48,7 +48,7 @@ You can join these groups and chats to discuss and ask TiDB related questions:
 
 - [Contributors Mailing list](https://lists.tidb.io/g/contributors)
 - [Slack Channel](https://slack.tidb.io/invite?team=tidb-community&channel=everyone&ref=pingcap-tidb)
-- [Chinese User Group](https://asktug.com)
+- [Chinese Forum](https://asktug.com)
```

Comment:
Can we change this to [Forum (Currently in Chinese)](https://asktug.com)?

Reply:
I don't think it is a good idea until we have a real English forum or at least a clear plan.

It is in fact a Chinese forum. Why do you want to make the change?

Reply:
You are right.